macros/common/create_external_schema.sql

```diff
@@ -4,8 +4,8 @@
 
 {% macro default__create_external_schema(source_node) %}
     {% set ddl %}
-        create schema if not exists {{ source_node.schema }}
+        create schema if not exists {{ source_node.database }}.{{ source_node.schema }}
```

Comment:
Without a database config on the source, we generate a ddl like;

```sql
create schema if not exists None.raw;
```

Maybe we should do something like:

```jinja
{% macro default__create_external_schema(source_node) %}
    {% set fqn -%}
        {%- if source_node.database -%}
            {{ source_node.database }}.{{ source_node.schema }}
        {%- else -%}
            {{ source_node.schema }}
        {%- endif -%}
    {%- endset %}

    {% set ddl %}
        create schema if not exists {{ fqn }}
    {% endset %}

    {{ return(ddl) }}
{% endmacro %}
```

Whatchu think? :)